.circleci/config.yml

```diff
@@ -160,11 +159,6 @@ jobs:
     steps:
     - fix-php-env
     - checkout
-    - run:
```

Comment:
Do you mind explaining why we stop generating the lock file? ```update --lock``` shouldn't update the core version at least.

Reply:
I thought we have that so we can increase version numbers inside the `admin_ui_support` module.